src/rpstest.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I would recommend naming this file the name of the function you are exporting - try to name your files so that its immediately apparent what's in them.